services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/geometria/GeometriatDaoImpl.kt

```diff
@@ -160,7 +165,8 @@ class GeometriatDaoImpl(private val jdbcOperations: JdbcOperations) : Geometriat
                             createdAt,
                             modifiedByUserId,
                             modifiedAt
-                        FROM Geometriat WHERE id = ?""".trimIndent(),
+                        FROM Geometriat WHERE id = ?"""
```

Comment:
The formatting changes were done by ktfmt. The tool version was updated some time ago in #483, and the formatting around multiline strings is different in the new version. The tool only re-formats files that have been edited, so the changes are propagated to source files when there are any edits to them.

I don't like this particular rule change, but it was necessary to update the ktfmt version, because the old version crashed with types like `ID & Any`.

Reply:
In practice, we have specified ratcheting rules in build.gradle.kts, which say that ktfmt should only look at files that have been changed from `origin/dev` branch. I.e. the dev branch in GitHub.
https://github.com/City-of-Helsinki/haitaton-backend/blob/efe9b7f1e58e51a2ea0cf73e4d6116e5c35b8988/services/hanke-service/build.gradle.kts#L44

Reply:
I thought it might be the reason. Thank you for clarifying it for me!